### PR TITLE
[naga-power] Naga Power Stategy

### PR DESCRIPTION
**Pull request by NuttakitDW**
Fixes # .

Changes proposed in this pull request:
- naga-power strategy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@ethersproject/strings": "^5.0.5",
     "@ethersproject/units": "^5.0.3",
     "@ethersproject/wallet": "^5.4.0",
+    "@snapshot-labs/snapshot.js": "^0.1.11",
```
Comment (ChaituVR) [inline package.json:23]: we are inside @snapshot-labs/snapshot.js... we don't need this.. you can remove it
```suggestion
```

Comment (ChaituVR) [reply package.json:23]: also, revert package-lock.json

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Looks like this is just a copy of `erc20-balance-of ` strategy. . you can use the same strategy in your space, with your own params

**Comment by bonustrack** [comment]
Closing this PR for now, new strategies must be added into this repository now: https://github.com/snapshot-labs/snapshot-strategies, feel free to reopen this PR if needed.

**Pull request closed without merging**